libfwupdplugin/fu-edid.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2023 Richard Hughes <richard@hughsie.com>
+// SPDX-License-Identifier: LGPL-2.1+
+
+#[repr(u8)]
+enum EdidDescriptorTag {
+    DisplayProductSerialNumber = 0xFF,
+    AlphanumericDataString = 0xFE,
+    DisplayRangeLimits = 0xFD,
+    DisplayProductName = 0xFC,
+    ColorPointData = 0xFB,
+    StandardTimingIdentifications = 0xFA,
+    DisplayColorManagementData = 0xF9,
+    CvtTimingCodes = 0xF8,
+    EstablishedTimings = 0xF7,
+    DummyDescriptor = 0x10,
+}
```

Comment:
don't you have the ordering backwards?

Reply:
I matched the order in the EDID spec -- I can flip it around if it's confusing.

Reply:
Oh if the spec is that way it's totally fine.